site/_apps/nextjs-hn.txt

```diff
@@ -0,0 +1,26 @@
+---
+weight: 14
+title:  'Next.js HN'
+github-title: 'chrisdwheatley/nextjs-hn'
+libraries:
+    - name: 'Next.js'
+    - name: 'Preact'
+module-bundling: 'Next.js with additional Webpack config'
+service-worker: 'Application & data caching with sw-precache'
+perfomance-patterns: 'In memory LRU cache, DNS prefetch, Preact compat alias in production'
+server-side-rendering: 'Yes'
+api: 'Node-hnapi (unofficial)'
+hosting: 'Digital Ocean & Cloudflare'
+authors:
+    - name: 'Chris'
+lighthouse: '91/100'
+interactive-em: '4.7s'
+interactive-faster-3g: '3.0s'
+lighthouse-link: https://www.webpagetest.org/result/170607_PW_1d454612c736a65ddff058cf16b8842e/
+wpt-em-link: https://www.webpagetest.org/result/170607_PW_1d454612c736a65ddff058cf16b8842e/
+wpt-faster-3g-link: https://www.webpagetest.org/result/170607_DY_400d1cc5db11ec9727404dc9cbc441e6/
```

Comment:
Could we re-run this one? We generally read the first interactive score from the attached Lighthouse report (viewable by clicking through on the Lighthouse score). This ensures that the scores noted in an entry also correspond to what scores you got during the run. 

![image](https://user-images.githubusercontent.com/110953/26901273-0c9ed19c-4b8a-11e7-9f5e-28ba64d9fc81.png)

Attempting to kick off a re-run here: https://www.webpagetest.org/result/170607_S2_17b6c30a3f2f8fb964d12dfeb5dba8a0/